Tests/SkillFunctionalTests/FileUpload/FileUploadTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using SkillFunctionalTests.Common;
+using TranscriptTestRunner;
+using TranscriptTestRunner.XUnit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SkillFunctionalTests.FileUpload
+{
+    [Trait("TestCategory", "FileUpload")]
+    public class FileUploadTests : ScriptTestBase
+    {
+        private readonly string _testScriptsFolder = Directory.GetCurrentDirectory() + @"/FileUpload/TestScripts";
+
+        public FileUploadTests(ITestOutputHelper output)
+            : base(output)
+        {
+        }
+
+        public static IEnumerable<object[]> TestCases()
+        {
+            var clientTypes = new List<ClientType> { ClientType.DirectLine };
```

Comment:
Hi @denscollo, I don't think this will compile after I merged https://github.com/microsoft/BotFramework-FunctionalTests/pull/309 (ClientType was removed there), can you update this branch to the latest from main and make sure it still runs?

Reply:
Hi @gabog, sure we'll update it and let you know when we have it

Reply:
Hi @gabog, branch updated. Thanks!